packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionLog.scss

```
@@ -87,3 +94,54 @@
         padding-left: 30px;
     }
 }
+
+.transaction-details {
+    background-color: $color-bg;
+    width: 100%;
+    height: 100%;
+    overflow: overlay;
+
+    &__header {
+        height: rem(20px);
+        font-size: rem(8px);
+        color: $color-transaction-group-header-text;
+        padding-left: rem(5px);
+        line-height: rem(12px);
+        display: flex;
+        align-items: center;
+        background-color: $color-transaction-group-header;
+        top: 0;
+        position: sticky;
+        z-index: 1;
+    }
+
+    &__close {
+        top: 0;
+        right: rem(10px);
+        position: absolute;
+    }
+
+    &__dynamic-height-item,
+    &__item {
+        position: relative;
+        padding: 0 rem(16px) 0 rem(8px);
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        border-top: 0.5px solid transparent;
+        border-top-color: $color-grey;
+        height: rem(50px);
+        word-break: break-all;
+    }
```

```suggestion
    &__item {
        height: rem(50px);
        padding-top: 0;
        padding-bottom: 0;
    }    
    &__dynamic-height-item,
    &__item {
        position: relative;
        padding-left: rem(16px);
        padding-right rem(8px);
        display: flex;
        flex-direction: column;
        justify-content: center;
        border-top: 0.5px solid transparent;
        border-top-color: $color-grey;
        word-break: break-all;
    }
```
I'm not a fan of specifying and overwritten properties like this (might be an irrational fear though).